wrims_v2/wrims_v2/src/test/test_heclib7/Test_heclib7.java

```diff
@@ -22,20 +23,33 @@
 
 public class Test_heclib7 extends TestCase {
 	
-	private int vHecFormat=6;
-	private String dvPath6="C:\\testHeclib7\\test_6.dss";
-	private String dvPath7="C:\\testHeclib7\\test_7.dss";
+	private String dvPath6="test\\data\\test_6.dss";
+	private String dvPath7="test\\data\\test_7.dss";
```

Comment:
I like to add some randomization to generated files' names so that we don't clash between sessions. Current timestamp is usually sufficient